nixos/modules/services/misc/workout-tracker.nix

```
@@ -0,0 +1,103 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib) types;
+  cfg = config.services.workout-tracker;
+in
+
+{
+  options = {
+    services.workout-tracker = {
+      enable = lib.mkEnableOption "A workout tracking web application for personal use (or family, friends), geared towards running and other GPX-based activities";
```

This will be prefixed with `Whether to`.

```suggestion
      enable = lib.mkEnableOption "workout tracking web application for personal use (or family, friends), geared towards running and other GPX-based activities";
```

---

Done